proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -61,6 +61,8 @@
       "            proxy and will have offline-mode UUIDs.",
       "- \"legacy\": Forward player IPs and UUIDs in a BungeeCord-compatible format. Use this if",
       "            you run servers using Minecraft 1.12 or lower.",
+      "- \"legacy_bungeeguard\": Forward player IPs and UUIDs in a BungeeCord-compatible format.",
```

Comment:
I'd prefer a description closer to this:

```
"bungeeguard": Forward player IPs and UUIDs in a format supported by
               the BungeeGuard plugin. Use this if you run servers
               using Minecraft 1.12 and below, are on a shared
               host, and would like to use BungeeGuard to provide
               additional security.
```

I'd prefer if this was made a bit more concise, though.